src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.PublicClr.approved.txt

```diff
@@ -846,6 +846,7 @@ namespace ServiceControl.MessageFailures
         public System.Collections.Generic.List<ServiceControl.MessageFailures.FailedMessage.ProcessingAttempt> ProcessingAttempts { get; set; }
         public ServiceControl.MessageFailures.FailedMessageStatus Status { get; set; }
         public string UniqueMessageId { get; set; }
+        public static string GetMessageIdFromDocumentId(string failedMessageDocumentId) { }
```

Comment:
Could this be internal? I'm not sure that we expose this outside of the SC anywhere anyway so it's probably not that important.

Reply:
:+1: Changed